napari/components/dims.py

```diff
@@ -23,44 +32,53 @@ class Dims(EventedModel):
     range : tuple of 3-tuple of float
-        List of tuples (min, max, step), one for each dimension. In a world
-        coordinates space. As with Python's `range` and `slice`, max is not
-        included.
-    current_step : tuple of int
-        Tuple of the slider position for each dims slider, in slider coordinates.
+        List of tuples (min, max, step), one for each dimension in world
+        coordinates space.
+    point : tuple of floats
+        Dims position in world coordinates for each dimension.
+    margin_left : tuple of floats
+        Left margin in world pixels of the slice for each dimension.
+    margin_right : tuple of floats
+        Right margin in world pixels of the slice for each dimension.
     order : tuple of int
         Tuple of ordering the dimensions, where the last dimensions are rendered.
     axis_labels : tuple of str
         Tuple of labels for each dimension.
+    last_used : int
+        Dimension which was last interacted with.
 
     Attributes
     ----------
     ndim : int
         Number of dimensions.
     ndisplay : int
         Number of displayed dimensions.
-    last_used : int
-        Dimension which was last used.
     range : tuple of 3-tuple of float
-        List of tuples (min, max, step), one for each dimension. In a world
-        coordinates space. As with Python's `range` and `slice`, max is not
-        included.
-    current_step : tuple of int
-        Tuple the slider position for each dims slider, in slider coordinates.
+        List of tuples (min, max, step), one for each dimension in world
+        coordinates space.
+    point : tuple of floats
+        Dims position in world coordinates for each dimension.
+    margin_left : tuple of floats
+        Left margin (=thickness) in world pixels of the slice for each dimension.
+    margin_right : tuple of floats
+        Right margin (=thickness) in world pixels of the slice for each dimension.
     order : tuple of int
         Tuple of ordering the dimensions, where the last dimensions are rendered.
     axis_labels : tuple of str
         Tuple of labels for each dimension.
+    last_used : int
+        Dimension which was last used.
+        Tuple the slider position for each dims slider, in world coordinates.
+    current_step : tuple of int
+        Current step for each dimension (same as point, but in slider coordinates).
     nsteps : tuple of int
         Number of steps available to each slider. These are calculated from
         the ``range``.
-    point : tuple of float
-        List of floats setting the current value of the range slider when in
-        POINT mode, one for each dimension. In a world coordinates space. These
-        are calculated from the ``current_step`` and ``range``.
+    thickness : tuple of floats
+        Thickness of the slice (sum of both margins) for each dimension in world coordinates.
+    point : tuple of floats
+        Center point of each span in world coordinates.
```

Comment:
This attribute is documented twice. I prefer this one as it's more descriptive.

Reply:
Though span needs clarifying/renaming given the current state of the PR.

Reply:
I don't think we should keep this one, we removed the concept of `span`, and more importantly the `point` is no longer guaranteed to be in the center.